microsoft/SqlNexus
# New xml rowsets for In-Memory OLTP and memory

Adding nem in-memory related tables to Nexus (related to Log Scout tasks 502 and 453).
Also included on this PR to old tables that were not being loaded: tbl_proccache_summary and tbl_proccache_pollution

How to test:

- Collect Log Scout, any perf scenario + memory, on an instance having in-memory tables. Suggestion: Sample databases from the following article can be used

https://learn.microsoft.com/en-us/sql/relational-databases/in-memory-oltp/sample-database-for-in-memory-oltp?view=sql-server-ver16

- Collect data from one instance 2014 and other 2016 or newer
- Load data into nexus.
-  Confirm that the following tables have been created

select * from dbo.tbl_proccache_summary

select * from dbo.tbl_proccache_pollution

select * from tbl_dm_db_xtp_index_stats

select * from tbl_dm_db_xtp_hash_index_stats

select * from tbl_dm_db_xtp_table_memory_stats

select * from tbl_dm_db_xtp_memory_consumers 

select * from tbl_dm_db_xtp_object_stats

select * from tbl_dm_xtp_system_memory_consumers

select * from tbl_dm_xtp_system_memory_consumers_summary

select * from tbl_dm_xtp_gc_stats

select * from tbl_dm_xtp_gc_queue_stats

select * from tbl_dm_db_xtp_transactions

select * from tbl_dm_xtp_transaction_stats
